specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -2368,15 +2427,34 @@
         "outboundRules": {
           "type": "object",
           "additionalProperties": {
+            "description": "Outbound Rule for the managed network of a machine learning workspace.",
             "$ref": "#/definitions/OutboundRule"
           },
           "x-nullable": true
```

Comment:
>   "x-nullable": true [](http://example.com/codeflow?start=7&length=21)

in general we discourage x-nullable, you might want to remove it to simplify, if its an API that isn't already GA